flatdata-rs/lib/src/test/test_generated.rs

```diff
@@ -0,0 +1,1310 @@
+// Do not edit: This code was generated by flatdata's generator.
```

Comment:
Avoid checking this one in? E.g. the feature tests generate code on-demand, which seems to work nicely.